sup3r/postprocessing/file_handling.py

```diff
@@ -46,7 +51,9 @@
             'pressure': {'scale_factor': 0.1,
                          'units': 'Pa',
                          'dtype': 'uint16',
-                         'chunks': (2000, 500)},
+                         'chunks': (2000, 500),
+                         'min': 0,
+                         'max': 100000},
```

Comment:
isnt this just 1 atm?

Reply:
yeah missed a 1. good catch

Reply:
No what i'm saying is that WRF regularly produces pressures > 1atm so it shouldnt be 101325 Pa (1atm) it should be more like 1.5e5 Pa

Reply:
Yeah I meant I missed a one to make it 1.1e5 but I'll increase it further

Reply:
Oh gotcha, i saw 1.01e5 and thought you were setting the ceiling at exactly 1atm. sounds good!